src/include/kdbopmphm.h

```diff
@@ -22,17 +22,25 @@
  * opmphmRatio should never be less than 1.
  *
  */
-#define OPMPHMTUPLE 7
+#define OPMPHMTUPLE 5
```

Comment:
For now the constant `OPMPHMTUPLE` is only a random number, after the evaluation of the benchmark results it will be final.